src/harness/collectionsImpl.ts

```diff
@@ -132,6 +132,7 @@ export class SortedMap<K, V> {
                 this._copyOnWrite = false;
             }
         }
+        return undefined;

```

Comment:
will all generators need to `return undefined` now? do you have an estimate of how much code that breaks?

Reply:
from reading type baselines, it looks like their return type changes to `any` instead of `undefined`

Reply:
This is specifically because this is an implementation of the native `Map<K, V>` interface, which is now specified to return `IterableIterator<T, BuiltinIteratorReturn>` (which is `IterableIterator<T, undefined>` under the new flag).